README.md

```diff
@@ -55,17 +55,16 @@ slither .
 ```
 ## 📜 Contract Addresses
 
- - For [Ethereum Mainnet](./docs/deployments/ethereum.md) - TODO.
- - For [Arbitrum One](./docs/deployments/rinkeby.md) - TODO.
+[Deployed Addresses](https://docs.fortress.finance/resources/smart-contracts).
```

Comment:
We do not document the deployer addresses in our docs yet. And I think the GLP compounder contract is being redeployed as well the next days.